### ISSUE TITLE
Attr_Expiration_Time not a Export_Defaults option

### ISSUE THREAD

**Issue by ajarr** [nfs-ganesha/nfs-ganesha#289]
At least in NFS-Ganesha v2.5.5 (the latest Ganesha v2.5 stable), the Attr_Expiration_Time option is not recognized as a Export_Defaults option, common to all exports, despite the following commit, 
https://github.com/nfs-ganesha/nfs-ganesha/commit/27ef33e44fce9ce904ee5a2650c96fd1fa0c57dd
You'll need to set it as a per export option.

When setting,
```
EXPORT_DEFAULTS {
        Attr_Expiration_Time = 0;
}

```
I get the following warning in ganesha.log
```
[main] config_errs_to_log :CONFIG :WARN :Config File (/etc/ganesha/ganesha.conf:10): Unknown parameter (Attr_Expiration_Time)
```

**Comment by dang** [comment]
So, this correct.  Currently, Attr_Expiration_Time is only in Export, not in Export_Defaults.  This can be moved from CONF_EXPORT_PARAMS to CONF_EXPORT_PERMS; but that would also put it into Client, where it would be ignored.  @ffilz what do you think?  Have it accepted but silently ignored in client?  Leave it as it currently is (but that means more difficult config)?  Handle it specially, outside of CONF_EXPORT_*?

**Comment by ffilz** [comment]
Gah, not reading code I WROTE closeley enough...

It can also be explicitly put in export_defaults_params which would allow it to be set in EXPORT_DEFAULTS but not CLIENT.

**Comment by ffilz** [comment]
I will make the change and push a patch.

**Comment by ajarr** [comment]
ffilz's commit that is supposed to fix this,
https://github.com/nfs-ganesha/nfs-ganesha/commit/e76c786d1f41d14b5bc3e4086136e11e93f04021

**Comment by ajarr** [comment]
The issue was fixed in V2.7-dev.9a and in v2.6.2 stable
